src/main/scala/com/yotpo/metorikku/metric/stepActions/dataQuality/ValidationRunner.scala

```diff
@@ -44,6 +56,35 @@ case class ValidationRunner() {
     }
   }
 
+  private def invokeSafely(func: ()=>Unit){
```

Comment:
It has a generic name but the log message is specific.
Wouldn't you think it's better to use try/catch within storeFailedDataFrame?
Do you think it's necessary for the custom use-case?

Reply:
You're right about the log message - I'll make it a more general one and move this method to Utils.
I didn't add it in the `storeFailedDataFrame` because I made the handler a parameter, and I don't want the block to fail for failing to handle a bad frame. Nevertheless, It is probably a good idea to add a `try/catch` block there too.